CRM/Event/BAO/ParticipantStatusType.php

```diff
@@ -117,6 +117,12 @@ public static function process($params) {
     $expiredStatuses = CRM_Event_PseudoConstant::participantStatus(NULL, "class = 'Negative'");
     $waitingStatuses = CRM_Event_PseudoConstant::participantStatus(NULL, "class = 'Waiting'");
 
+    // Ignore Pending pay later and incomplete transactions status.
+    $pendingStatuses = array_diff($pendingStatuses, [
+      'Pending from pay later',
+      'Pending from incomplete transaction',
+    ]);
```

Comment:
tech notes to self:
* I checked and the function does return `name`'s not `label`'s, so this would be ok.
* this process() function only seems to be used by the update participant status job